### PR TITLE
bee mutatoxins

### PR DESCRIPTION
**Pull request by triplezeta**
## About The Pull Request

Original PR: https://github.com/BeeStation/BeeStation-Hornet/pull/617

EDIT: and https://github.com/BeeStation/BeeStation-Hornet/pull/831

## Why It's Good For The Game

its on the trello board 

## Changelog
:cl:
add: Ported Beestation mutation toxin recipes
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
[PORT BEE MUTATION TOXINS](https://trello.com/c/74j3Pymu/38-port-bee-mutation-toxins)

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -2156,3 +2200,10 @@
 	color = "#623301"
 	taste_mult = 1.2
 
+
+/datum/reagent/liquidadamantine
+	name = "Liquid Adamantine"
+	description = "A legengary lifegiving metal liquified."
+	color = "#10cca6" //RGB: 16, 204, 166
+	taste_description = "lifegiiving metal"
+	can_synth = FALSE
```
Comment (MarkSuckerberg) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:2209]:
```suggestion
	can_synth = FALSE
	
```

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -2156,3 +2200,11 @@
...
+
```
Comment (MarkSuckerberg) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:2210]:
```suggestion

```

**Comment by triplezeta** [comment]
that's what fucking fixed it-

grill

**Comment by TheNeoGamer42** [comment]
wait what what broke YOUR PR hMM? A line that added a fucking battery to a never used gun that apparently is actually just used in the AI turrets is what broke mine for the fucking bot even though it didnt on my end

**Comment by triplezeta** [comment]
a missing newline that, when added, still broke the PR, but when removed, made the PR work?

**Comment by TheNeoGamer42** [comment]
w h a t

**Comment by triplezeta** [comment]
yeah exactly

**Comment by triplezeta** [comment]
@MarkSuckerberg 
merge_my_pr

**Comment by TheNeoGamer42** [comment]
how about instead of merging the PR someone tell me how to speedrun making felinid mutation toxin so I can make everyone want to fucking kill themselves, then merge it

**Comment by triplezeta** [comment]
the recipe is on the beestation wiki

**Comment by TheNeoGamer42** [comment]
we need a wiki for boomer, chemistry is fucking confusing here since theres no wiki and I dont fucking know which of the goddamn wikis recipes WILL work

**Comment by triplezeta** [comment]
hang on, let me just ask the headmins when webspace will exist for the Nth time

**Comment by TheNeoGamer42** [comment]
ask them at gunpoint this time, specifically with the L666 FUCKYOUINATOR, also should I give it a different name?

**Pull request merged**